outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -247,6 +255,73 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
         );
     }
 
+    private getNoCompanyInsightsSection = () => {
+        let NoInsightsSection = null;
+        const company = this.state.company as Company;
+        if (company.isEmpty()) {
+            if (this.props.partner.isAddedToDatabase() && !this.enriched) {
+                NoInsightsSection = (
+                    <div>v
+                        <div>
+                            {_t("No company attached to this contact")}
+                        </div>
+                        <div className="odoo-secondary-button" style={{margin: "8px auto auto auto"}}
+                             onClick={this.enrichCompanyForPartnerRequest}>
+                            {_t("Create a Company")}
+                        </div>
+                    </div>
+                );
+            } else {
+                if (!this.context.isConnected()) {
+                    NoInsightsSection = (
+                        <div>
+                            {_t("No company linked to this contact could be enriched")}
+                        </div>
+                    );
+                } else {
+                    NoInsightsSection = (
+                        <div>
+                            {_t("No company linked to this contact could be enriched or found in Odoo")}
+                        </div>
+                    );
+                }
+            }
+        } else {
+            if (this.enriched) {
+                NoInsightsSection = (<div>
+                    {_t("No insights where found for this company")}
+                </div>);
+            } else {
+                NoInsightsSection = (
+                    <div>
+                        {_t("No insights for this company")}
+                        <div className='odoo-secondary-button' style={{margin: "8px auto auto auto"}}
+                             onClick={this.updateCompanyRequest}>
+                            Enrich Company
+                        </div>
+                    </div>
+                );
+            }
+        }
+
+        return (
+            <div className={"muted-text"}
+                 style={{textAlign: "center", padding: "8px 16px 16px 16px", fontSize: "14px"}}>
```

Comment:
Would be nice if all inlined-style could instead go into a css class.

Reply:
Yes, inlined-styles moved to a css class.